posts/2024-03-29-ArgoCD-Configuration-Drift.adoc

```diff
@@ -0,0 +1,161 @@
+To get the default Admin password run the following command:
+```
+kubectl get secret argocd-initial-admin-secret -n argocd -o jsonpath='{.data.password}' | base64 -d
+```
+To access the ArgoCD UI, go to the route of the `argocd-server` and log with the `admin` username and password.
+[.img_border_light]
+image::/img/blog/configuration-drift-1.png[Argo Login,width=70%,align="center"]
+
+Once logged in, you should see the ArgoCD landing page similar to the screenshot below:
+[.img_border_light]
+image::/img/blog/configuration-drift-2.png[Argo Home,width=70%,align="center"]
+
+[#setup_gitops_repo]
+== Setting up a GitOps Repository for your Application
+Now that you've successfully installed ArgoCD and gained access to its console, the next step is to set up a GitHub repository for ArgoCD to synchronize with.  In our example we have created a deployment directory in the application code repo which we will point ArgoCD to for deployment.  
+
+_NOTE: In our experience we found it better to separate the deployment configuration into a separate repository from the application code. This aligns with recommended best practices to maintain separation between configuration repositories and code repositories. Keeping these separate ensures that infrastructure changes are decoupled from application changes, allowing for more granular control, better audit trails, and reduced risk of unintended side-effects.  The reasons why can be found in https://argo-cd.readthedocs.io/en/stable/user-guide/best_practices/[ArgoCD's documentation on best practices]. However, for this particular example, because it is just a small demo application, we have put the deployment configuration with the application code._
+
+For the purposes of this blog, we'll use the Daytrader7 sample application. This application is built around the concept of an online stock trading system, and the Daytrader7 https://github.com/WASdev/sample.daytrader7#readme[README] provides a comprehensive overview of this application. In this example, we have a `deploy` directory within our existing repository to house the deployment configuration.  This directory contains the manifests for full application deployment as well as annotations telling ArgoCD which order they should be deployed in.  
+
+To begin, navigate to GitHub and fork this repository: https://github.com/WASdev/sample.daytrader7. Use your forked repository when configuring ArgoCD in the following steps.
+
+In this example, we have a pre-built container image for DayTrader sample application pushed to Docker Hub that is used in the deployment files deploy/daytrader7-deploy.yaml.  For more information on how to containerize your Open Liberty application check out our interactive guide: https://openliberty.io/guides/docker.html[Using Docker containers to develop microservices].
+
+[#deploy_app_with_gitops]
+== Deploying a Java application with GitOps and ArgoCD
```

Comment:
I'd revise this section to use numbered steps, possibly nesting steps for the form under a single parent step. Also, use **bold** for any UI elements in the text